saleor/payment/gateways/braintree/__init__.py

```diff
@@ -32,6 +32,7 @@
 
 
 def get_customer_data(payment_information: PaymentData) -> Dict:
+    ''' Provides customer info, use only for new customer creation '''
```

Comment:
Our style is like this:

```suggestion
    """Provides customer info, use only for new customer creation."""
```